pkg/eventservice/event_scanner.go

```diff
@@ -470,7 +470,7 @@ func (m *eventMerger) appendRemainingDDLs(endTs uint64) []event.Event {
 		m.ddlIndex++
 	}
 
-	events = append(events, pevent.NewResolvedEvent(endTs, m.dispatcherID, m.epoch))
+	events = append(events, pevent.NewResolvedEvent(endTs, m.dispatcherID, m.epoch, false))
```

Comment:
why don't we need to consider redo here?

Reply:
Yes, it will be considered by sendResolvedTs